variables.ini.dist

```diff
@@ -122,6 +122,7 @@ OPENSTACK_DEFAULT_ROUTER = ; openstack_router
 OPENSTACK_USER_ROLE = ; defaultMemberRole
 OPENSTACK_ADMIN_ROLE = ; admin
 ATMOSPHERE_SUPERUSER = ; atmo_superuser
+ATMOSPHERE_SECRET_SEED = ; (Random value:) 9oe5q7cp714zmmjl51
```

Comment:
Random Value, A-Z and 0-9 required. Any length of string is fine.

I will add this to the comments.. I dont think we need a generator at this level.